region/crateria/east/The Moat.json

```diff
@@ -473,93 +456,85 @@
       },
       "requires": [
-        "canShinechargeMovementComplex",
-        {"shinespark": {"frames": 23}}
+        "canShinechargeMovementComplex"
       ],
-      "exitCondition": {
-        "leaveWithSpark": {}
-      },
-      "unlocksDoors": [
-        {"types": ["super"], "requires": []},
-        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
-      ],
-      "note": [
-        "After gaining a shinecharge, run back to the door to make space for a running jump to the left, firing a shot just before landing on the pedestal.",
-        "Activate the shinespark either just before landing on the pedestal or just afterward (depending on the vertical position needed for the spark)."
-      ]
+      "collectsItems": [3],
+      "endsWithShineCharge": true
     },
     {
-      "link": [2, 1],
-      "name": "Hero Shot Spark",
+      "link": [1, 3],
+      "name": "Hero Shot Spark Through Item",
       "entranceCondition": {
         "comeInShinecharged": {
-          "framesRequired": 25
+          "framesRequired": 40
         }
       },
       "requires": [
         "canShinechargeMovementTricky",
         "canHeroShot",
-        {"shinespark": {"frames": 39}}
-      ],
-      "exitCondition": {
-        "leaveWithSpark": {}
-      },
-      "unlocksDoors": [
-        {"types": ["super"], "requires": []},
-        {"types": ["missiles", "powerbomb"], "requires": ["never"]}
+        "canHorizontalShinespark",
+        {"shinespark": {"frames": 38}}
       ],
+      "clearsObstacles": ["B", "C"],
+      "collectsItems": [3],
       "flashSuitChecked": true,
       "note": [
-        "Run, fire a shot mid-air, jump, and activate a mid-air shinespark to the left.",
+        "Run and jump, fire a shot mid-air, and activate the shinespark wind-up.",
+        "Wait until the shot is at a specific position before sparking.",
         "Sparking too late will cause the shot to despawn before reaching the door.",
         "Sparking too early will cause Samus to bonk the door as it will not yet be open.",
         "A beam shot, Missile, or Super can be used."
+      ],
+      "devNote": "Sparking in bottom position means the item will be collected along the way."
+    },
+    {
+      "link": [2, 1],
+      "name": "Base",
+      "requires": [
+        {"or": [
+          "Grapple",
+          "SpaceJump"
+        ]}
+      ],
+      "collectsItems": [3],
+      "note": "This avoids collecting the item.",
+      "devNote": "Several other strats would also be possible for avoiding the item."
+    },
+    {
+      "link": [2, 1],
+      "name": "Pass Below",
+      "requires": [
+        "h_canBombThings"
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Gravity",
+      "requires": [
+        "Gravity",
+        {"or": [
+          "canWalljump",
+          "canGravityJump",
+          {"and": [
+            "canSpringBallJumpMidAir",
+            "HiJump"
+          ]}
+        ]}
+      ],
+      "collectsItems": [3]
```

Comment:
Some of these don't need to collect the item

Reply:
Ah good catch. I moved the strat from 2->3 to 2->1 but forgot to remove the collectsItems. Kyle checked (and I double-checked) that all of the cases can be done without collecting the item